src/main/java/edu/ucsb/cs156/example/repositories/UCSBDateRepository.java

```diff
@@ -0,0 +1,12 @@
+package edu.ucsb.cs156.example.repositories;
+
+import edu.ucsb.cs156.example.entities.UCSBDate;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+
+@Repository
+public interface UCSBDateRepository extends CrudRepository<UCSBDate, Long> {
+  Iterable<UCSBDate> findAllByquarterYYYYQ(String quarterYYYYQ);
```

Comment:
Doesn't this need to be `findAllByQuarterYYYYQ` (capital Q)?

Reply:
Yes, good catch